daemon/graphdriver/quota/errors.go

```diff
@@ -0,0 +1,6 @@
+package quota
+
+import "errors"
+
+// ErrQuotaNotSupported indicates if were found the FS didn't have projects quotas available
+var ErrQuotaNotSupported = errors.New("Filesystem does not support, or has not enabled quotas")
```

Comment:
Can we make sure this error implements one of api/errdefs?

Reply:
Which one do you think it should be?

Reply:
ErrNotImplemented?

Reply:
Or ErrUnavailable

Reply:
NotImplemented sounds good.